[bug] cmake_find_package_multi: frameworks not propagated on MacOS

<!--
  Please don't forget to update the issue title.
  Include all applicable information to help us reproduce your problem.

  To help us debug your issue please explain:
-->

### Environment Details (include every applicable attribute)
  * Operating System+version: MacOS
  * Compiler+version: apple-clang 9.4
  * Conan version: 1.28.1
  * Python version: 3.7.6

### Steps to reproduce (Include if Applicable)

- clone https://github.com/SpaceIm/conan-libgeotiff
- checkout 38abc7aa75e9fd48fd87caae8ccffee3d67b4b3a
- run `conan create . libgeotiff/1.6.0@`

Workaround:
- in `test_package` replace imported target (provided by config file generated by `cmake_find_package_multi`), by `${CONAN_LIBS}`: https://github.com/SpaceIm/conan-libgeotiff/compare/38abc7aa75e9...7334db46e7e4, then it works and we see frameworks (coming from `libcurl` transitive dependency) linked in `test_package`

### Logs (Executed commands with output) (Include/Attach if Applicable)

<details><summary>Click to expand log</summary>

```
 >> Build: 3/4

****************************************************************************************************

+-------------------------+

| Profile                 |

|-------------------------|

| include(default)        |

|                         |

| [settings]              |

| arch=x86_64             |

| build_type=Release      |

| compiler=apple-clang    |

| compiler.version=9.1    |

| [options]               |

| libgeotiff:shared=False |

| [env]                   |

|                         |

| [build_requires]        |

+-------------------------+

[conan_create]

 >> Calling 'conan create'

+-----------------+--------------------------------------------------------------------------------------------------+

| Configuration   | value                                                                                            |

|-----------------+--------------------------------------------------------------------------------------------------|

| name            | libgeotiff                                                                                       |

| version         | 1.6.0                                                                                            |

| user            | SpaceIm                                                                                          |

| channel         | testing                                                                                          |

| build_modes     | ['missing']                                                                                      |

| profile_name    | /var/folders/nz/vv4_9tw56nv9k3tkvyszvwg80000gn/T/tmpupmovmv6conan_package_tools_profiles/profile |

+-----------------+--------------------------------------------------------------------------------------------------+

Exporting package recipe

libgeotiff/1.6.0@SpaceIm/testing exports: File 'conandata.yml' found. Exporting it...

libgeotiff/1.6.0@SpaceIm/testing exports: Copied 1 '.yml' file: conandata.yml

libgeotiff/1.6.0@SpaceIm/testing exports_sources: Copied 1 '.txt' file: CMakeLists.txt

libgeotiff/1.6.0@SpaceIm/testing exports_sources: Copied 2 '.patch' files: fix-cmake-1.5.1.patch, fix-cmake-1.6.0.patch

libgeotiff/1.6.0@SpaceIm/testing: The stored package has not changed

libgeotiff/1.6.0@SpaceIm/testing: Exported revision: 0367d26dede60ef04226b07ca9fc9bae

Configuration:

[settings]

arch=x86_64

arch_build=x86_64

build_type=Release

compiler=apple-clang

compiler.libcxx=libc++

compiler.version=9.1

os=Macos

os_build=Macos

[options]

libgeotiff:shared=False

[build_requires]

[env]

libgeotiff/1.6.0@SpaceIm/testing (test package): Installing package

Requirements

    jbig/20160605 from 'conan-center' - Cache

    libcurl/7.71.0 from 'conan-center' - Cache

    libgeotiff/1.6.0@SpaceIm/testing from local cache - Cache

    libjpeg/9d from 'conan-center' - Cache

    libtiff/4.1.0 from 'conan-center' - Cache

    libwebp/1.1.0 from 'conan-center' - Cache

    proj/7.1.0 from 'conan-center' - Cache

    sqlite3/3.32.3 from 'conan-center' - Cache

    xz_utils/5.2.4 from 'conan-center' - Cache

    zlib/1.2.11 from 'conan-center' - Cache

    zstd/1.4.5 from 'conan-center' - Cache

Packages

    jbig/20160605:af96383df8ed9748d2eed48fb04099c5914a718f - Cache

    libcurl/7.71.0:c524ca20330d4dcedff6342b243b52f8309c04c9 - Cache

    libgeotiff/1.6.0@SpaceIm/testing:8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7 - Build

    libjpeg/9d:af96383df8ed9748d2eed48fb04099c5914a718f - Cache

    libtiff/4.1.0:1ec604129ec3b8252cd0dc5909a62344db0bcd37 - Cache

    libwebp/1.1.0:b2489a85e468d4b7346e677154155459063fe1fe - Cache

    proj/7.1.0:1917467295b8e93b0afec186b01efef58607cd1f - Cache

    sqlite3/3.32.3:7c6ac9fa75013ba73c1e1fd9735f04c4e82a8b7b - Cache

    xz_utils/5.2.4:af96383df8ed9748d2eed48fb04099c5914a718f - Cache

    zlib/1.2.11:af96383df8ed9748d2eed48fb04099c5914a718f - Cache

    zstd/1.4.5:af96383df8ed9748d2eed48fb04099c5914a718f - Cache

Installing (downloading, building) binaries...

jbig/20160605: Already installed!

jbig/20160605: Appending PATH environment variable: /Users/travis/.conan/data/jbig/20160605/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/bin

libjpeg/9d: Already installed!

libwebp/1.1.0: Already installed!

sqlite3/3.32.3: Already installed!

sqlite3/3.32.3: Appending PATH env var with : /Users/travis/.conan/data/sqlite3/3.32.3/_/_/package/7c6ac9fa75013ba73c1e1fd9735f04c4e82a8b7b/bin

xz_utils/5.2.4: Already installed!

zlib/1.2.11: Already installed!

zstd/1.4.5: Already installed!

libcurl/7.71.0: Already installed!

libtiff/4.1.0: Already installed!

proj/7.1.0: Already installed!

proj/7.1.0: Appending PROJ_LIB environment variable: /Users/travis/.conan/data/proj/7.1.0/_/_/package/1917467295b8e93b0afec186b01efef58607cd1f/res

proj/7.1.0: Appending PATH environment variable: /Users/travis/.conan/data/proj/7.1.0/_/_/package/1917467295b8e93b0afec186b01efef58607cd1f/bin

libgeotiff/1.6.0@SpaceIm/testing: Copying sources to build folder

libgeotiff/1.6.0@SpaceIm/testing: Building your package in /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/build/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake created conanbuildinfo.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created PROJConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created PROJTarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created PROJTargets.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created PROJConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created TIFFConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created TIFFConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created TIFFTargets.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created TIFFTarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created SQLite3ConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created SQLite3Target-release.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created SQLite3Targets.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created SQLite3Config.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created CURLConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created CURLTarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created CURLTargets.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created CURLConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created ZLIBConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created ZLIBConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created ZLIBTargets.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created ZLIBTarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created LibLZMAConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created LibLZMAConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created LibLZMATargets.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created LibLZMATarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created JPEGConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created JPEGConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created JPEGTargets.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created JPEGTarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created jbigConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created jbigConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created jbigTargets.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created jbigTarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created zstdConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created zstdTarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created zstdTargets.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created zstdConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created libwebpConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created libwebpConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created libwebpTargets.cmake

libgeotiff/1.6.0@SpaceIm/testing: Generator cmake_find_package_multi created libwebpTarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing: Calling build()

-- The C compiler identification is AppleClang 9.1.0.9020039

-- The CXX compiler identification is AppleClang 9.1.0.9020039

-- Detecting C compiler ABI info

-- Detecting C compiler ABI info - done

-- Check for working C compiler: /Applications/Xcode-9.4.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc - skipped

-- Detecting C compile features

-- Detecting C compile features - done

-- Detecting CXX compiler ABI info

-- Detecting CXX compiler ABI info - done

-- Check for working CXX compiler: /Applications/Xcode-9.4.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - skipped

-- Detecting CXX compile features

-- Detecting CXX compile features - done

-- Conan: called by CMake conan helper

-- Conan: called inside local cache

-- Conan: Adjusting output directories

-- Conan: Using cmake global configuration

-- Conan: Adjusting default RPATHs Conan policies

-- Conan: Adjusting language standard

-- Conan: Adjusting fPIC flag (ON)

-- Looking for include file stdio.h

-- Looking for include file stdio.h - found

-- Looking for include file stdlib.h

-- Looking for include file stdlib.h - found

-- Looking for include file string.h

-- Looking for include file string.h - found

-- Looking for include file strings.h

-- Looking for include file strings.h - found

-- Library proj found /Users/travis/.conan/data/proj/7.1.0/_/_/package/1917467295b8e93b0afec186b01efef58607cd1f/lib/libproj.a

-- Found: /Users/travis/.conan/data/proj/7.1.0/_/_/package/1917467295b8e93b0afec186b01efef58607cd1f/lib/libproj.a

-- Library proj found /Users/travis/.conan/data/proj/7.1.0/_/_/package/1917467295b8e93b0afec186b01efef58607cd1f/lib/libproj.a

-- Found: /Users/travis/.conan/data/proj/7.1.0/_/_/package/1917467295b8e93b0afec186b01efef58607cd1f/lib/libproj.a

-- Library sqlite3 found /Users/travis/.conan/data/sqlite3/3.32.3/_/_/package/7c6ac9fa75013ba73c1e1fd9735f04c4e82a8b7b/lib/libsqlite3.a

-- Found: /Users/travis/.conan/data/sqlite3/3.32.3/_/_/package/7c6ac9fa75013ba73c1e1fd9735f04c4e82a8b7b/lib/libsqlite3.a

-- Library sqlite3 found /Users/travis/.conan/data/sqlite3/3.32.3/_/_/package/7c6ac9fa75013ba73c1e1fd9735f04c4e82a8b7b/lib/libsqlite3.a

-- Found: /Users/travis/.conan/data/sqlite3/3.32.3/_/_/package/7c6ac9fa75013ba73c1e1fd9735f04c4e82a8b7b/lib/libsqlite3.a

-- Library tiffxx found /Users/travis/.conan/data/libtiff/4.1.0/_/_/package/1ec604129ec3b8252cd0dc5909a62344db0bcd37/lib/libtiffxx.a

-- Found: /Users/travis/.conan/data/libtiff/4.1.0/_/_/package/1ec604129ec3b8252cd0dc5909a62344db0bcd37/lib/libtiffxx.a

-- Library tiff found /Users/travis/.conan/data/libtiff/4.1.0/_/_/package/1ec604129ec3b8252cd0dc5909a62344db0bcd37/lib/libtiff.a

-- Found: /Users/travis/.conan/data/libtiff/4.1.0/_/_/package/1ec604129ec3b8252cd0dc5909a62344db0bcd37/lib/libtiff.a

-- Library z found /Users/travis/.conan/data/zlib/1.2.11/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libz.a

-- Found: /Users/travis/.conan/data/zlib/1.2.11/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libz.a

-- Library lzma found /Users/travis/.conan/data/xz_utils/5.2.4/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/liblzma.a

-- Found: /Users/travis/.conan/data/xz_utils/5.2.4/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/liblzma.a

-- Library jpeg found /Users/travis/.conan/data/libjpeg/9d/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libjpeg.a

-- Found: /Users/travis/.conan/data/libjpeg/9d/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libjpeg.a

-- Library jbig found /Users/travis/.conan/data/jbig/20160605/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libjbig.a

-- Found: /Users/travis/.conan/data/jbig/20160605/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libjbig.a

-- Library zstd found /Users/travis/.conan/data/zstd/1.4.5/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libzstd.a

-- Found: /Users/travis/.conan/data/zstd/1.4.5/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libzstd.a

-- Library zstd found /Users/travis/.conan/data/zstd/1.4.5/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libzstd.a

-- Found: /Users/travis/.conan/data/zstd/1.4.5/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libzstd.a

-- Library webpmux found /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebpmux.a

-- Found: /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebpmux.a

-- Library webpdemux found /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebpdemux.a

-- Found: /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebpdemux.a

-- Library webpdecoder found /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebpdecoder.a

-- Found: /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebpdecoder.a

-- Library webp found /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebp.a

-- Found: /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebp.a

-- Library curl found /Users/travis/.conan/data/libcurl/7.71.0/_/_/package/c524ca20330d4dcedff6342b243b52f8309c04c9/lib/libcurl.a

-- Found: /Users/travis/.conan/data/libcurl/7.71.0/_/_/package/c524ca20330d4dcedff6342b243b52f8309c04c9/lib/libcurl.a

-- Library curl found /Users/travis/.conan/data/libcurl/7.71.0/_/_/package/c524ca20330d4dcedff6342b243b52f8309c04c9/lib/libcurl.a

-- Found: /Users/travis/.conan/data/libcurl/7.71.0/_/_/package/c524ca20330d4dcedff6342b243b52f8309c04c9/lib/libcurl.a

-- Dependency ZLIB already found

-- Library tiffxx found /Users/travis/.conan/data/libtiff/4.1.0/_/_/package/1ec604129ec3b8252cd0dc5909a62344db0bcd37/lib/libtiffxx.a

-- Skipping already existing target: CONAN_LIB::TIFF_tiffxx_RELEASE

-- Found: /Users/travis/.conan/data/libtiff/4.1.0/_/_/package/1ec604129ec3b8252cd0dc5909a62344db0bcd37/lib/libtiffxx.a

-- Library tiff found /Users/travis/.conan/data/libtiff/4.1.0/_/_/package/1ec604129ec3b8252cd0dc5909a62344db0bcd37/lib/libtiff.a

-- Skipping already existing target: CONAN_LIB::TIFF_tiff_RELEASE

-- Found: /Users/travis/.conan/data/libtiff/4.1.0/_/_/package/1ec604129ec3b8252cd0dc5909a62344db0bcd37/lib/libtiff.a

-- Dependency ZLIB already found

-- Dependency LibLZMA already found

-- Dependency JPEG already found

-- Dependency jbig already found

-- Dependency zstd already found

-- Dependency libwebp already found

-- Generating geo_config.h header

-- Generating geo_config.h header - done

-- Configuring done

-- Generating done

CMake Warning:

  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY

    CMAKE_INSTALL_BINDIR

    CMAKE_INSTALL_DATAROOTDIR

    CMAKE_INSTALL_INCLUDEDIR

    CMAKE_INSTALL_LIBDIR

    CMAKE_INSTALL_LIBEXECDIR

    CMAKE_INSTALL_OLDINCLUDEDIR

    CMAKE_INSTALL_SBINDIR

-- Build files have been written to: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/build/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/build_subfolder

Scanning dependencies of target geotiff_library

[ 12%] Building C object source_subfolder/libgeotiff/CMakeFiles/geotiff_library.dir/geo_extra.c.o

[ 12%] Building C object source_subfolder/libgeotiff/CMakeFiles/geotiff_library.dir/cpl_serv.c.o

[ 18%] Building C object source_subfolder/libgeotiff/CMakeFiles/geotiff_library.dir/geo_free.c.o

[ 25%] Building C object source_subfolder/libgeotiff/CMakeFiles/geotiff_library.dir/geo_get.c.o

[ 31%] Building C object source_subfolder/libgeotiff/CMakeFiles/geotiff_library.dir/geo_names.c.o

[ 37%] Building C object source_subfolder/libgeotiff/CMakeFiles/geotiff_library.dir/geo_new.c.o

[ 43%] Building C object source_subfolder/libgeotiff/CMakeFiles/geotiff_library.dir/geo_normalize.c.o

[ 50%] Building C object source_subfolder/libgeotiff/CMakeFiles/geotiff_library.dir/geo_print.c.o

[ 56%] Building C object source_subfolder/libgeotiff/CMakeFiles/geotiff_library.dir/geo_set.c.o

[ 62%] Building C object source_subfolder/libgeotiff/CMakeFiles/geotiff_library.dir/geo_simpletags.c.o

[ 68%] Building C object source_subfolder/libgeotiff/CMakeFiles/geotiff_library.dir/geo_tiffp.c.o

[ 75%] Building C object source_subfolder/libgeotiff/CMakeFiles/geotiff_library.dir/geo_trans.c.o

[ 81%] Building C object source_subfolder/libgeotiff/CMakeFiles/geotiff_library.dir/geo_write.c.o

[ 87%] Building C object source_subfolder/libgeotiff/CMakeFiles/geotiff_library.dir/geotiff_proj4.c.o

[ 93%] Building C object source_subfolder/libgeotiff/CMakeFiles/geotiff_library.dir/libxtiff/xtiff.c.o

[100%] Linking C static library ../../lib/libgeotiff.a

[100%] Built target geotiff_library

libgeotiff/1.6.0@SpaceIm/testing: Package '8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7' built

libgeotiff/1.6.0@SpaceIm/testing: Build folder /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/build/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7

libgeotiff/1.6.0@SpaceIm/testing: Generated conaninfo.txt

libgeotiff/1.6.0@SpaceIm/testing: Generated conanbuildinfo.txt

libgeotiff/1.6.0@SpaceIm/testing: Generating the package

libgeotiff/1.6.0@SpaceIm/testing: Package folder /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7

libgeotiff/1.6.0@SpaceIm/testing: Calling package()

[100%] Built target geotiff_library

Install the project...

-- Install configuration: "Release"

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/doc/AUTHORS

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/doc/ChangeLog

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/doc/COPYING

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/doc/INSTALL

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/doc/LICENSE

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/doc/README

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/doc/README_BIN

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/doc/README.WIN

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/share/man/man1/applygeo.1

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/share/man/man1/geotifcp.1

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/share/man/man1/listgeo.1

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/geotiff.h

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/geotiffio.h

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/geovalues.h

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/geonames.h

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/geokeys.h

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/geo_config.h

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/geo_tiffp.h

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/geo_keyp.h

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/geo_normalize.h

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/cpl_serv.h

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/geo_simpletags.h

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/epsg_datum.inc

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/epsg_gcs.inc

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/epsg_pm.inc

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/epsg_units.inc

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/geo_ctrans.inc

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/epsg_ellipse.inc

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/epsg_pcs.inc

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/epsg_proj.inc

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/epsg_vertcs.inc

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/geokeys.inc

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/geokeys_v1_1.inc

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include/xtiffio.h

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/lib/libgeotiff.a

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/share/cmake/GeoTIFF/geotiff-config.cmake

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/share/cmake/GeoTIFF/geotiff-config-version.cmake

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/share/cmake/GeoTIFF/geotiff-depends.cmake

-- Installing: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/share/cmake/GeoTIFF/geotiff-depends-release.cmake

libgeotiff/1.6.0@SpaceIm/testing package(): Packaged 12 '.h' files

libgeotiff/1.6.0@SpaceIm/testing package(): Packaged 11 '.inc' files

libgeotiff/1.6.0@SpaceIm/testing package(): Packaged 1 '.a' file: libgeotiff.a

libgeotiff/1.6.0@SpaceIm/testing package(): Packaged 1 file: LICENSE

libgeotiff/1.6.0@SpaceIm/testing: Package '8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7' created

libgeotiff/1.6.0@SpaceIm/testing: Created package revision 5a272417e304a1e89487264131e73af8

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake created conanbuildinfo.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator txt created conanbuildinfo.txt

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created GeoTIFFConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created GeoTIFFTarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created GeoTIFFTargets.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created GeoTIFFConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created PROJConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created PROJTarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created PROJTargets.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created PROJConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created TIFFConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created TIFFConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created TIFFTargets.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created TIFFTarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created SQLite3ConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created SQLite3Target-release.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created SQLite3Targets.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created SQLite3Config.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created CURLConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created CURLTarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created CURLTargets.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created CURLConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created ZLIBConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created ZLIBConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created ZLIBTargets.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created ZLIBTarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created LibLZMAConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created LibLZMAConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created LibLZMATargets.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created LibLZMATarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created JPEGConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created JPEGConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created JPEGTargets.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created JPEGTarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created jbigConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created jbigConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created jbigTargets.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created jbigTarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created zstdConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created zstdTarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created zstdTargets.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created zstdConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created libwebpConfigVersion.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created libwebpConfig.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created libwebpTargets.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generator cmake_find_package_multi created libwebpTarget-release.cmake

libgeotiff/1.6.0@SpaceIm/testing (test package): Generated conaninfo.txt

libgeotiff/1.6.0@SpaceIm/testing (test package): Generated graphinfo

Using lockfile: '/Users/travis/build/SpaceIm/conan-libgeotiff/test_package/build/cd42d2846ceb758a7aa2902302439760f3d98a7d/conan.lock'

Using cached profile from lockfile

libgeotiff/1.6.0@SpaceIm/testing (test package): Calling build()

-- The C compiler identification is AppleClang 9.1.0.9020039

-- Detecting C compiler ABI info

-- Detecting C compiler ABI info - done

-- Check for working C compiler: /Applications/Xcode-9.4.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc - skipped

-- Detecting C compile features

-- Detecting C compile features - done

-- Conan: called by CMake conan helper

-- Conan: Adjusting output directories

-- Conan: Using cmake global configuration

-- Conan: Adjusting default RPATHs Conan policies

-- Conan: Adjusting language standard

-- This project seems to be plain C, using 'AppleClang' compiler

-- Conan: C++ stdlib: libc++

-- Library geotiff found /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/lib/libgeotiff.a

-- Found: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/lib/libgeotiff.a

-- Library geotiff found /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/lib/libgeotiff.a

-- Found: /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/lib/libgeotiff.a

-- Library tiffxx found /Users/travis/.conan/data/libtiff/4.1.0/_/_/package/1ec604129ec3b8252cd0dc5909a62344db0bcd37/lib/libtiffxx.a

-- Found: /Users/travis/.conan/data/libtiff/4.1.0/_/_/package/1ec604129ec3b8252cd0dc5909a62344db0bcd37/lib/libtiffxx.a

-- Library tiff found /Users/travis/.conan/data/libtiff/4.1.0/_/_/package/1ec604129ec3b8252cd0dc5909a62344db0bcd37/lib/libtiff.a

-- Found: /Users/travis/.conan/data/libtiff/4.1.0/_/_/package/1ec604129ec3b8252cd0dc5909a62344db0bcd37/lib/libtiff.a

-- Library z found /Users/travis/.conan/data/zlib/1.2.11/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libz.a

-- Found: /Users/travis/.conan/data/zlib/1.2.11/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libz.a

-- Library lzma found /Users/travis/.conan/data/xz_utils/5.2.4/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/liblzma.a

-- Found: /Users/travis/.conan/data/xz_utils/5.2.4/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/liblzma.a

-- Library jpeg found /Users/travis/.conan/data/libjpeg/9d/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libjpeg.a

-- Found: /Users/travis/.conan/data/libjpeg/9d/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libjpeg.a

-- Library jbig found /Users/travis/.conan/data/jbig/20160605/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libjbig.a

-- Found: /Users/travis/.conan/data/jbig/20160605/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libjbig.a

-- Library zstd found /Users/travis/.conan/data/zstd/1.4.5/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libzstd.a

-- Found: /Users/travis/.conan/data/zstd/1.4.5/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libzstd.a

-- Library zstd found /Users/travis/.conan/data/zstd/1.4.5/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libzstd.a

-- Found: /Users/travis/.conan/data/zstd/1.4.5/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libzstd.a

-- Library webpmux found /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebpmux.a

-- Found: /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebpmux.a

-- Library webpdemux found /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebpdemux.a

-- Found: /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebpdemux.a

-- Library webpdecoder found /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebpdecoder.a

-- Found: /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebpdecoder.a

-- Library webp found /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebp.a

-- Found: /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebp.a

-- Library proj found /Users/travis/.conan/data/proj/7.1.0/_/_/package/1917467295b8e93b0afec186b01efef58607cd1f/lib/libproj.a

-- Found: /Users/travis/.conan/data/proj/7.1.0/_/_/package/1917467295b8e93b0afec186b01efef58607cd1f/lib/libproj.a

-- Library proj found /Users/travis/.conan/data/proj/7.1.0/_/_/package/1917467295b8e93b0afec186b01efef58607cd1f/lib/libproj.a

-- Found: /Users/travis/.conan/data/proj/7.1.0/_/_/package/1917467295b8e93b0afec186b01efef58607cd1f/lib/libproj.a

-- Library sqlite3 found /Users/travis/.conan/data/sqlite3/3.32.3/_/_/package/7c6ac9fa75013ba73c1e1fd9735f04c4e82a8b7b/lib/libsqlite3.a

-- Found: /Users/travis/.conan/data/sqlite3/3.32.3/_/_/package/7c6ac9fa75013ba73c1e1fd9735f04c4e82a8b7b/lib/libsqlite3.a

-- Library sqlite3 found /Users/travis/.conan/data/sqlite3/3.32.3/_/_/package/7c6ac9fa75013ba73c1e1fd9735f04c4e82a8b7b/lib/libsqlite3.a

-- Found: /Users/travis/.conan/data/sqlite3/3.32.3/_/_/package/7c6ac9fa75013ba73c1e1fd9735f04c4e82a8b7b/lib/libsqlite3.a

-- Dependency TIFF already found

-- Library curl found /Users/travis/.conan/data/libcurl/7.71.0/_/_/package/c524ca20330d4dcedff6342b243b52f8309c04c9/lib/libcurl.a

-- Found: /Users/travis/.conan/data/libcurl/7.71.0/_/_/package/c524ca20330d4dcedff6342b243b52f8309c04c9/lib/libcurl.a

-- Library curl found /Users/travis/.conan/data/libcurl/7.71.0/_/_/package/c524ca20330d4dcedff6342b243b52f8309c04c9/lib/libcurl.a

-- Found: /Users/travis/.conan/data/libcurl/7.71.0/_/_/package/c524ca20330d4dcedff6342b243b52f8309c04c9/lib/libcurl.a

-- Dependency ZLIB already found

-- Configuring done

-- Generating done

CMake Warning:

  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY

-- Build files have been written to: /Users/travis/build/SpaceIm/conan-libgeotiff/test_package/build/cd42d2846ceb758a7aa2902302439760f3d98a7d

/Users/travis/.pyenv/versions/3.7.6/envs/conan/lib/python3.7/site-packages/cmake/data/CMake.app/Contents/bin/cmake -S/Users/travis/build/SpaceIm/conan-libgeotiff/test_package -B/Users/travis/build/SpaceIm/conan-libgeotiff/test_package/build/cd42d2846ceb758a7aa2902302439760f3d98a7d --check-build-system CMakeFiles/Makefile.cmake 0

/Users/travis/.pyenv/versions/3.7.6/envs/conan/lib/python3.7/site-packages/cmake/data/CMake.app/Contents/bin/cmake -E cmake_progress_start /Users/travis/build/SpaceIm/conan-libgeotiff/test_package/build/cd42d2846ceb758a7aa2902302439760f3d98a7d/CMakeFiles /Users/travis/build/SpaceIm/conan-libgeotiff/test_package/build/cd42d2846ceb758a7aa2902302439760f3d98a7d//CMakeFiles/progress.marks

/Applications/Xcode-9.4.1.app/Contents/Developer/usr/bin/make  -f CMakeFiles/Makefile2 all

/Applications/Xcode-9.4.1.app/Contents/Developer/usr/bin/make  -f CMakeFiles/test_package.dir/build.make CMakeFiles/test_package.dir/depend

cd /Users/travis/build/SpaceIm/conan-libgeotiff/test_package/build/cd42d2846ceb758a7aa2902302439760f3d98a7d && /Users/travis/.pyenv/versions/3.7.6/envs/conan/lib/python3.7/site-packages/cmake/data/CMake.app/Contents/bin/cmake -E cmake_depends "Unix Makefiles" /Users/travis/build/SpaceIm/conan-libgeotiff/test_package /Users/travis/build/SpaceIm/conan-libgeotiff/test_package /Users/travis/build/SpaceIm/conan-libgeotiff/test_package/build/cd42d2846ceb758a7aa2902302439760f3d98a7d /Users/travis/build/SpaceIm/conan-libgeotiff/test_package/build/cd42d2846ceb758a7aa2902302439760f3d98a7d /Users/travis/build/SpaceIm/conan-libgeotiff/test_package/build/cd42d2846ceb758a7aa2902302439760f3d98a7d/CMakeFiles/test_package.dir/DependInfo.cmake --color=

Scanning dependencies of target test_package

/Applications/Xcode-9.4.1.app/Contents/Developer/usr/bin/make  -f CMakeFiles/test_package.dir/build.make CMakeFiles/test_package.dir/build

[ 50%] Building C object CMakeFiles/test_package.dir/test_package.c.o

/Applications/Xcode-9.4.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DCURL_STATICLIB=1 -DLIBJPEG_STATIC -DLZMA_API_STATIC -isystem /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/include -isystem /Users/travis/.conan/data/proj/7.1.0/_/_/package/1917467295b8e93b0afec186b01efef58607cd1f/include -isystem /Users/travis/.conan/data/libtiff/4.1.0/_/_/package/1ec604129ec3b8252cd0dc5909a62344db0bcd37/include -isystem /Users/travis/.conan/data/sqlite3/3.32.3/_/_/package/7c6ac9fa75013ba73c1e1fd9735f04c4e82a8b7b/include -isystem /Users/travis/.conan/data/libcurl/7.71.0/_/_/package/c524ca20330d4dcedff6342b243b52f8309c04c9/include -isystem /Users/travis/.conan/data/zlib/1.2.11/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/include -isystem /Users/travis/.conan/data/xz_utils/5.2.4/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/include -isystem /Users/travis/.conan/data/libjpeg/9d/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/include -isystem /Users/travis/.conan/data/jbig/20160605/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/include -isystem /Users/travis/.conan/data/zstd/1.4.5/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/include -isystem /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/include -m64 -O3 -DNDEBUG  -arch x86_64 -isysroot /Applications/Xcode-9.4.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk -DLIBJPEG_STATIC -DLZMA_API_STATIC -DCURL_STATICLIB=1 -o CMakeFiles/test_package.dir/test_package.c.o -c /Users/travis/build/SpaceIm/conan-libgeotiff/test_package/test_package.c

[100%] Linking C executable bin/test_package

/Users/travis/.pyenv/versions/3.7.6/envs/conan/lib/python3.7/site-packages/cmake/data/CMake.app/Contents/bin/cmake -E cmake_link_script CMakeFiles/test_package.dir/link.txt --verbose=1

/Applications/Xcode-9.4.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc   -m64 -O3 -DNDEBUG  -arch x86_64 -isysroot /Applications/Xcode-9.4.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names CMakeFiles/test_package.dir/test_package.c.o -o bin/test_package   -L/Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/lib  -L/Users/travis/.conan/data/proj/7.1.0/_/_/package/1917467295b8e93b0afec186b01efef58607cd1f/lib  -L/Users/travis/.conan/data/libtiff/4.1.0/_/_/package/1ec604129ec3b8252cd0dc5909a62344db0bcd37/lib  -L/Users/travis/.conan/data/sqlite3/3.32.3/_/_/package/7c6ac9fa75013ba73c1e1fd9735f04c4e82a8b7b/lib  -L/Users/travis/.conan/data/libcurl/7.71.0/_/_/package/c524ca20330d4dcedff6342b243b52f8309c04c9/lib  -L/Users/travis/.conan/data/zlib/1.2.11/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib  -L/Users/travis/.conan/data/xz_utils/5.2.4/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib  -L/Users/travis/.conan/data/libjpeg/9d/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib  -L/Users/travis/.conan/data/jbig/20160605/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib  -L/Users/travis/.conan/data/zstd/1.4.5/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib  -L/Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib  /Users/travis/.conan/data/libgeotiff/1.6.0/SpaceIm/testing/package/8bd4aafdf81f9f6928a6ee1db2060bd1a568b0c7/lib/libgeotiff.a /Users/travis/.conan/data/proj/7.1.0/_/_/package/1917467295b8e93b0afec186b01efef58607cd1f/lib/libproj.a /Users/travis/.conan/data/libtiff/4.1.0/_/_/package/1ec604129ec3b8252cd0dc5909a62344db0bcd37/lib/libtiffxx.a /Users/travis/.conan/data/libtiff/4.1.0/_/_/package/1ec604129ec3b8252cd0dc5909a62344db0bcd37/lib/libtiff.a /Users/travis/.conan/data/xz_utils/5.2.4/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/liblzma.a /Users/travis/.conan/data/libjpeg/9d/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libjpeg.a /Users/travis/.conan/data/jbig/20160605/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libjbig.a /Users/travis/.conan/data/zstd/1.4.5/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libzstd.a /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebpmux.a /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebpdemux.a /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebpdecoder.a /Users/travis/.conan/data/libwebp/1.1.0/_/_/package/b2489a85e468d4b7346e677154155459063fe1fe/lib/libwebp.a -lc++ /Users/travis/.conan/data/sqlite3/3.32.3/_/_/package/7c6ac9fa75013ba73c1e1fd9735f04c4e82a8b7b/lib/libsqlite3.a /Users/travis/.conan/data/libcurl/7.71.0/_/_/package/c524ca20330d4dcedff6342b243b52f8309c04c9/lib/libcurl.a /Users/travis/.conan/data/zlib/1.2.11/_/_/package/af96383df8ed9748d2eed48fb04099c5914a718f/lib/libz.a 

Undefined symbols for architecture x86_64:

  "_SSLClose", referenced from:

      _Curl_sectransp_shutdown in libcurl.a(libcurl_la-sectransp.o)

      _Curl_sectransp_close in libcurl.a(libcurl_la-sectransp.o)

  "_SSLCopyPeerTrust", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

      _sectransp_connect_step2 in libcurl.a(libcurl_la-sectransp.o)

      _verify_cert in libcurl.a(libcurl_la-sectransp.o)

  "_SSLCreateContext", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SSLGetBufferedReadSize", referenced from:

      _Curl_sectransp_data_pending in libcurl.a(libcurl_la-sectransp.o)

  "_SSLGetNegotiatedCipher", referenced from:

      _sectransp_connect_step2 in libcurl.a(libcurl_la-sectransp.o)

  "_SSLGetNegotiatedProtocolVersion", referenced from:

      _sectransp_connect_step2 in libcurl.a(libcurl_la-sectransp.o)

  "_SSLGetNumberSupportedCiphers", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SSLGetSessionState", referenced from:

      _Curl_sectransp_check_cxn in libcurl.a(libcurl_la-sectransp.o)

  "_SSLGetSupportedCiphers", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SSLHandshake", referenced from:

      _sectransp_connect_step2 in libcurl.a(libcurl_la-sectransp.o)

  "_SSLRead", referenced from:

      _sectransp_recv in libcurl.a(libcurl_la-sectransp.o)

  "_SSLSetCertificate", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SSLSetConnection", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SSLSetEnableCertVerify", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SSLSetEnabledCiphers", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SSLSetIOFuncs", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SSLSetPeerDomainName", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SSLSetPeerID", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SSLSetProtocolVersionMax", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SSLSetProtocolVersionMin", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SSLSetSessionOption", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SSLWrite", referenced from:

      _sectransp_send in libcurl.a(libcurl_la-sectransp.o)

  "_SecCertificateCopyCommonName", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SecCertificateCopyLongDescription", referenced from:

      _CopyCertSubject in libcurl.a(libcurl_la-sectransp.o)

  "_SecCertificateCreateWithData", referenced from:

      _append_cert_to_array in libcurl.a(libcurl_la-sectransp.o)

  "_SecCertificateGetTypeID", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SecIdentityCopyCertificate", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SecIdentityCreateWithCertificate", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SecItemCopyMatching", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SecItemExport", referenced from:

      _sectransp_connect_step2 in libcurl.a(libcurl_la-sectransp.o)

  "_SecItemImport", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SecPolicyCreateSSL", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SecTrustCopyPublicKey", referenced from:

      _sectransp_connect_step2 in libcurl.a(libcurl_la-sectransp.o)

  "_SecTrustEvaluate", referenced from:

      _verify_cert in libcurl.a(libcurl_la-sectransp.o)

  "_SecTrustGetCertificateAtIndex", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SecTrustGetCertificateCount", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_SecTrustSetAnchorCertificates", referenced from:

      _verify_cert in libcurl.a(libcurl_la-sectransp.o)

  "_SecTrustSetAnchorCertificatesOnly", referenced from:

      _verify_cert in libcurl.a(libcurl_la-sectransp.o)

  "_kSecAttrLabel", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_kSecClass", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_kSecClassIdentity", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_kSecImportItemIdentity", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_kSecMatchLimit", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_kSecMatchLimitAll", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_kSecMatchPolicy", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

  "_kSecReturnRef", referenced from:

      _sectransp_connect_common in libcurl.a(libcurl_la-sectransp.o)

ld: symbol(s) not found for architecture x86_64

clang: error: linker command failed with exit code 1 (use -v to see invocation)

make[2]: *** [bin/test_package] Error 1

make[1]: *** [CMakeFiles/test_package.dir/all] Error 2

make: *** [all] Error 2

Traceback (most recent call last):

  File "/Users/travis/.pyenv/versions/conan/lib/python3.7/site-packages/conans/errors.py", line 34, in conanfile_exception_formatter

    yield

  File "/Users/travis/.pyenv/versions/conan/lib/python3.7/site-packages/conans/client/conanfile/build.py", line 16, in run_build_method

    conanfile.build()

  File "/Users/travis/build/SpaceIm/conan-libgeotiff/test_package/conanfile.py", line 12, in build

    cmake.build()

  File "/Users/travis/.pyenv/versions/conan/lib/python3.7/site-packages/conans/client/build/cmake.py", line 310, in build

    self._build(args, build_dir, target)

  File "/Users/travis/.pyenv/versions/conan/lib/python3.7/site-packages/conans/client/build/cmake.py", line 345, in _build

    self._run(command)

  File "/Users/travis/.pyenv/versions/conan/lib/python3.7/site-packages/conans/client/build/cmake.py", line 262, in _run

    self._conanfile.run(command)

  File "/Users/travis/.pyenv/versions/conan/lib/python3.7/site-packages/conans/model/conan_file.py", line 290, in run

    raise ConanException("Error %d while executing %s" % (retcode, command))

conans.errors.ConanException: Error 2 while executing cmake --build '/Users/travis/build/SpaceIm/conan-libgeotiff/test_package/build/cd42d2846ceb758a7aa2902302439760f3d98a7d' '--' '-j2'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):

  File "build.py", line 7, in <module>

    builder.run()

  File "/Users/travis/.pyenv/versions/conan/lib/python3.7/site-packages/cpt/packager.py", line 594, in run

    self.run_builds(base_profile_name=base_profile_name)

  File "/Users/travis/.pyenv/versions/conan/lib/python3.7/site-packages/cpt/packager.py", line 687, in run_builds

    r.run()

  File "/Users/travis/.pyenv/versions/conan/lib/python3.7/site-packages/cpt/runner.py", line 134, in run

    lockfile=self._lockfile)

  File "/Users/travis/.pyenv/versions/conan/lib/python3.7/site-packages/conans/client/conan_api.py", line 94, in wrapper

    return f(api, *args, **kwargs)

  File "/Users/travis/.pyenv/versions/conan/lib/python3.7/site-packages/conans/client/conan_api.py", line 378, in create

    test_build_folder, test_folder, conanfile_path, recorder=recorder)

  File "/Users/travis/.pyenv/versions/conan/lib/python3.7/site-packages/conans/client/cmd/create.py", line 77, in create

    recorder=recorder)

  File "/Users/travis/.pyenv/versions/conan/lib/python3.7/site-packages/conans/client/cmd/test.py", line 40, in install_build_and_test

    install_folder=test_build_folder, test=reference)

  File "/Users/travis/.pyenv/versions/conan/lib/python3.7/site-packages/conans/client/cmd/build.py", line 48, in cmd_build

    run_build_method(conan_file, app.hook_manager, conanfile_path=conanfile_path)

  File "/Users/travis/.pyenv/versions/conan/lib/python3.7/site-packages/conans/client/conanfile/build.py", line 16, in run_build_method

    conanfile.build()

  File "/Users/travis/.pyenv/versions/3.7.6/lib/python3.7/contextlib.py", line 130, in __exit__

    self.gen.throw(type, value, traceback)

  File "/Users/travis/.pyenv/versions/conan/lib/python3.7/site-packages/conans/errors.py", line 40, in conanfile_exception_formatter

    raise ConanExceptionInUserConanfileMethod(msg)

conans.errors.ConanExceptionInUserConanfileMethod: libgeotiff/1.6.0@SpaceIm/testing (test package): Error in build() method, line 12

	cmake.build()

	ConanException: Error 2 while executing cmake --build '/Users/travis/build/SpaceIm/conan-libgeotiff/test_package/build/cd42d2846ceb758a7aa2902302439760f3d98a7d' '--' '-j2'

Traceback (most recent call last):

  File "/Users/travis/.pyenv/versions/conan/bin/bincrafters-package-tools", line 11, in <module>

    load_entry_point('bincrafters-package-tools==0.27.5', 'console_scripts', 'bincrafters-package-tools')()

  File "/Users/travis/.pyenv/versions/3.7.6/envs/conan/lib/python3.7/site-packages/bincrafters/cli.py", line 35, in cli

    run(sys.argv[1:])

  File "/Users/travis/.pyenv/versions/3.7.6/envs/conan/lib/python3.7/site-packages/bincrafters/cli.py", line 27, in run

    run_autodetect()

  File "/Users/travis/.pyenv/versions/3.7.6/envs/conan/lib/python3.7/site-packages/bincrafters/build_autodetect.py", line 38, in run_autodetect

    subprocess.run(["python",  "{}".format(custom_build_py_path)], check=True)

  File "/Users/travis/.pyenv/versions/3.7.6/lib/python3.7/subprocess.py", line 512, in run

    output=stdout, stderr=stderr)

subprocess.CalledProcessError: Command '['python', 'build.py']' returned non-zero exit status 1.

The command "bincrafters-package-tools --auto" exited with 1.
```

</details>

--- a/conans/client/generators/cmake_find_package_multi.py
+++ b/conans/client/generators/cmake_find_package_multi.py
@@ -135,7 +135,7 @@ set_property(TARGET {name}::{name}
         ########## COMPONENT {{ comp_name }} FIND LIBRARIES & FRAMEWORKS / DYNAMIC VARS #############
 
         set({{ pkg_name }}_{{ comp_name }}_FRAMEWORKS_FOUND_{{ build_type }} "")
-        conan_find_apple_frameworks({{ pkg_name }}_{{ comp_name }}_FRAMEWORKS_FOUND_{{ build_type }} "{{ '${'+pkg_name+'_'+comp_name+'_FRAMEWORKS}' }}" "{{ '${'+pkg_name+'_'+comp_name+'_FRAMEWORK_DIRS}' }}")
+        conan_find_apple_frameworks({{ pkg_name }}_{{ comp_name }}_FRAMEWORKS_FOUND_{{ build_type }} "{{ '${'+pkg_name+'_'+comp_name+'_FRAMEWORKS_'+build_type+'}' }}" "{{ '${'+pkg_name+'_'+comp_name+'_FRAMEWORK_DIRS_'+build_type+'}' }}")
 
         set({{ pkg_name }}_{{ comp_name }}_LIB_TARGETS_{{ build_type }} "")
         set({{ pkg_name }}_{{ comp_name }}_NOT_USED_{{ build_type }} "")

--- a/conans/test/functional/generators/cmake_apple_frameworks_test.py
+++ b/conans/test/functional/generators/cmake_apple_frameworks_test.py
@@ -456,3 +456,135 @@ class CMakeAppleOwnFrameworksTestCase(unittest.TestCase):
             # Check we are using the framework
             link_txt = t.load(os.path.join('CMakeFiles', 'test_package.dir', 'link.txt'))
             self.assertIn("-framework Foundation", link_txt)
+
+    def component_test(self):
+        conanfile_py = textwrap.dedent("""
+from conans import ConanFile, CMake, tools
+
+
+class HelloConan(ConanFile):
+    name = "hello"
+    description = "example"
+    topics = ("conan",)
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.example.com"
+    license = "MIT"
+    exports_sources = ["hello.cpp", "hello.h", "CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    _cmake = None
+
+    def source(self):
+        pass
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "HELLO"
+        self.cpp_info.names["cmake_find_package_multi"] = "HELLO"
+        self.cpp_info.components["libhello"].names["cmake_find_package"] = "libhello"
+        self.cpp_info.components["libhello"].names["cmake_find_package_multi"] = "libhello"
+
+        self.cpp_info.components["libhello"].libs = ["hello"]
+        self.cpp_info.components["libhello"].frameworks.extend(["CoreFoundation"])
+        """)
+        hello_cpp = textwrap.dedent("""
+#include <CoreFoundation/CoreFoundation.h>
+
+void hello_api()
+{
+    CFTypeRef keys[] = {CFSTR("key")};
+    CFTypeRef values[] = {CFSTR("value")};
+    CFDictionaryRef dict = CFDictionaryCreate(kCFAllocatorDefault, keys, values, sizeof(keys) / sizeof(keys[0]), &kCFTypeDictionaryKeyCallBacks, &kCFTypeDictionaryValueCallBacks);
+    if (dict)
+        CFRelease(dict);
+}
+        """)
+        hello_h = textwrap.dedent("""
+void hello_api();
+        """)
+        cmakelists_txt = textwrap.dedent("""
+cmake_minimum_required(VERSION 2.8)
+
+project(hello)
+
+include(GNUInstallDirs)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+file(GLOB SOURCES *.cpp)
+file(GLOB HEADERS *.h)
+
+add_library(${PROJECT_NAME} ${SOURCES} ${HEADERS})
+
+set_target_properties(${PROJECT_NAME} PROPERTIES PUBLIC_HEADER ${HEADERS})
+install(TARGETS ${PROJECT_NAME}
+    RUNTIME DESTINATION bin
+    LIBRARY DESTINATION lib
+    ARCHIVE DESTINATION lib
+    PUBLIC_HEADER DESTINATION include)
+        """)
+        tp_conanfile_py = textwrap.dedent("""
+import os
+from conans import ConanFile, CMake, tools
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            bin_path = os.path.join("bin", "test_package")
+            self.run(bin_path, run_environment=True)
+        """)
+        tp_test_package_cpp = textwrap.dedent("""
+#include "hello.h"
+
+int main()
+{
+    hello_api();
+}
+        """)
+        tp_cmakelists_txt = textwrap.dedent("""
+cmake_minimum_required(VERSION 2.8)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(HELLO REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} HELLO::libhello)
+        """)
+        t = TestClient()
+        t.save({'conanfile.py': conanfile_py,
+                'hello.cpp': hello_cpp,
+                'hello.h': hello_h,
+                'CMakeLists.txt': cmakelists_txt,
+                'test_package/conanfile.py': tp_conanfile_py,
+                'test_package/CMakeLists.txt': tp_cmakelists_txt,
+                'test_package/test_package.cpp': tp_test_package_cpp})
+        t.run("create . hello/1.0@")

